doc/develop/test/twister.rst

```diff
@@ -36,17 +36,39 @@ These also affect the outputs of ``--testcase-report`` and
 
 To run the script in the local tree, follow the steps below:
 
-::
+.. tabs::
+
+   .. group-tab:: Ubuntu
```

Comment:
> Since this is applicable to all Linux distros and not specifically Ubuntu, it would be better to just specify Linux here:
> 
> ```diff
> -   .. group-tab:: Ubuntu
> +   .. group-tab:: Linux
> ```

yep, I updated.